ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_HasType_User()
+        {
+            //Arrange
+            Article _article = new Article();
+            User _user = new User("Tommy");
+            //Act
+            _article.Autor = _user;
```

Comment:
The `Autor` property has strong type so this test will always return true and does not matter which exactly are `_article` and `_user`. You need to test that property is set when you provide a `User` instancy by constructor.

Additionally can be checked that `Title` property is null.

```suggestion
            //Arrange
            User _user = new User("Tommy");
            //Act
            Article _article = new Article(_user);
```